.github/workflows/e2e.yml

```diff
@@ -0,0 +1,93 @@
+name: E2E Cypress
```

Comment:
can you name the file explictly cypress.yml?

Reply:
Done